common/components/BalanceSidebar/AccountInfo.tsx

```diff
@@ -80,9 +80,17 @@ export default class AccountInfo extends React.Component<Props, State> {
                 className="AccountInfo-list-item-clickable mono wrap"
                 onClick={this.toggleShowLongBalance}
               >
-                {this.state.showLongBalance
-                  ? balance ? balance.toString() : '???'
-                  : balance ? formatNumber(balance.amount) : '???'}
+                {this.state.showLongBalance ? (
+                  balance.isPending ? (
+                    <Spinner size="x1" />
+                  ) : (
+                    balance.amount.toString()
+                  )
+                ) : balance.isPending ? (
+                  <Spinner size="x1" />
```

Comment:
Spinner shows up twice here, might be simpler to make that the opening ternary check, ala:

```
balance.isPending ? (spinner) : (other checks)
```